### ISSUE TITLE
chromate reductase activity (GO:0034567)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-ontology#28058]
id: GO:0034567
name: chromate reductase activity
namespace: molecular_function
def: "Catalysis of the reaction: chromate = chromium (III)." [UM-BBD_reactionID:r0884]
xref: EC:1.97.1.-
xref: UM-BBD_reactionID:r0884
is_a: GO:0016491 ! oxidoreductase activity

This has 3 EXP annotations to E.coli nfsA, chrR, nemA.
I can't see any matching EC/RHEA entry.

But the partial EC (which corresponds to a higher level grouping term) should be removed.

**Comment by sjm41** [comment]
Looking at this one again....

There are only three annotations to this term, to E.coli genes authored by EcoCyc:

UniProtKB:P0AGE6 	chrR 	PMID:14766567 	
UniProtKB:P17117 	nfsA 	PMID:15250887 	
UniProtKB:P77258 	nemA 	PMID:23527133 	

But if I go to EcoCyc, none of those genes are annotated to catalyse this "chromate = chromium (III)" reaction.

I think the reaction is equivalent to one/both of these RHEA entries:
RHEA:44368 (EcoCyc:RXN0-3381)
Cr(6+) + 2 NADPH + O2 = Cr(3+) + superoxide + 2 NADP(+) + 2 H(+)


RHEA:44372  (EcoCyc:RXN0-3381)
Cr(6+) + 2 NADH + O2 = Cr(3+) + superoxide + 2 NAD(+) + 2 H(+)


EcoCyc:RXN0-3381 is:
Cr6+ + 2 NAD(P)H + dioxygen → Cr3+ + 2 NAD(P)+ + superoxide + 2 H+
It's annotated to one gene (yieF, so a different gene to the 3 above) with the comment "The enzyme may catalyze this reaction in vitro, but this reaction is not considered to be physiologically relevant."


So, taken together, I think this means that we should obsolete GO:0034567 chromate reductase activity.

@amandamackie Do you agree?

Thanks

**Comment by amandamackie** [comment]
Yes agreed. The EcoCyc GO annotations to this activity are incorrect. The chromate reductase activity associated with the E. coli enzymes is promiscuous.

**Comment by sjm41** [comment]
Thanks @amandamackie 

@raymond91125 - please obsolete this term.
